pyqt-apps/siriushla/as_di_bpms/settings.py

```diff
@@ -491,3 +494,357 @@ def _setupFMCPICOWidget(self):
+                            self, f"IA-{area_str}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                            )
+                    elif bo_num in afcs_pvs.BO_N1:
+                        area_index = afcs_pvs.BO_N1.index(bo_num) + 1
+                        area_str = f"{area_index:02}"
+                        lab_pv = QLabel(k, alignment=Qt.AlignCenter)
+                        slab_pv = SiriusLabel(
+                            self, f"IA-{area_str}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                            )
+                    elif bo_num in afcs_pvs.BO_N2:
+                        area_index = afcs_pvs.BO_N2.index(bo_num) + 1
+                        area_str = f"{area_index:02}"
+                        lab_pv = QLabel(k, alignment=Qt.AlignCenter)
+                        slab_pv = SiriusLabel(
+                            self, f"IA-{area_str}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                            )
+                else:
+                    lab_pv = QLabel(k, alignment=Qt.AlignCenter)
+                    slab_pv = SiriusLabel(
+                        self, f"IA-{self.bpm[3:5]}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                        )
+                lay_sensor.addWidget(lab_pv, row, 0)
+                lay_sensor.addWidget(slab_pv, row, 1)
+
+            elif 'Cte' in pv_name:
+                if self.bpm[:2] == 'TS' or self.bpm[:2] == 'TB':
+                    led_pv = SiriusLedAlert(
+                        self, f"IA-20RaBPMTL:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                        )
+                elif self.bpm[:2] == 'BO':
+                    bo_num = int(self.bpm.split('-')[1][:2])
+                    if bo_num in afcs_pvs.BO_N:
+                        area_index = afcs_pvs.BO_N.index(bo_num) + 1
+                        area_str = f"{area_index:02}"
+                        led_pv = SiriusLedAlert(
+                            self, f"IA-{area_str}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                            )
+                    elif bo_num in afcs_pvs.BO_N1:
+                        area_index = afcs_pvs.BO_N1.index(bo_num) + 1
+                        area_str = f"{area_index:02}"
+                        led_pv = SiriusLedAlert(
+                            self, f"IA-{area_str}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                            )
+                    elif bo_num in afcs_pvs.BO_N2:
+                        area_index = afcs_pvs.BO_N2.index(bo_num) + 1
+                        area_str = f"{area_index:02}"
+                        led_pv = SiriusLedAlert(
+                            self, f"IA-{area_str}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
+                            )
+                else:
+                    led_pv = SiriusLedAlert(
+                        self, f"IA-{self.bpm[3:5]}RaBPM:{afcs_pvs.DIS}-AMC-{amc_number}:{pv_name}"
```

Comment:
same suggestions of using `SiriusPVName`

maybe we can also have a function to convert from device name to the `subsection`, once this logic is used in several parts of the code